www/src/components/__tests__/banner.js

```diff
@@ -3,7 +3,7 @@ import { render } from "react-testing-library"
 
 import Banner from "../banner"
 
-const matchers = [`Watch now`, `Register now`]
+const matchers = [`Watch now`, `Register now`, `Gatsby Preview`]
```

Comment:
the old matchers are unused?

Reply:
Yeah that's because they've bounced back between before/after webinars. When the text of the banner changes we won't have to update the test again. It's not exactly the best or most useful test anyway.

Reply:
maybe better solution are rotating banners?